content/webapp/pages/events.tsx

```diff
@@ -112,7 +112,7 @@ const EventsPage: FC<Props> = props => {
             },
           ]}
           paginatedResults={convertedPaginatedResults}
-          paginationRoot={`events${period ? `/${period}` : ''}`}
+          paginationRoot={`/events${period ? `/${period}` : ''}`}
```

Comment:
I've double checked the pagination and canonical URLs; pushing this slash up here doesn't break anything.